cache/src/main/kotlin/com/dropbox/android/external/cache4/Cache.kt

```diff
@@ -153,20 +160,9 @@ internal class CacheBuilderImpl : Cache.Builder {
         this.clock = clock
     }
 
+    @ExperimentalTime
     override fun <K : Any, V : Any> build(): Cache<K, V> {
-        val effectiveExpireAfterWrite = if (expireAfterWriteNanos == UNSET_LONG) {
-            DEFAULT_EXPIRATION_NANOS
-        } else {
-            expireAfterWriteNanos
-        }
-
-        val effectiveExpireAfterAccess = if (expireAfterAccessNanos == UNSET_LONG) {
-            DEFAULT_EXPIRATION_NANOS
-        } else {
-            expireAfterAccessNanos
-        }
-
-        val effectiveMaxSize = if (expireAfterWriteNanos == 0L || expireAfterAccessNanos == 0L) {
+        val effectiveMaxSize = if (expireAfterWriteDuration == Duration.ZERO || expireAfterAccessDuration == Duration.ZERO) {
```

Comment:
This behavior was originally intended for testing but I don't think we end up relying on this in our tests so I'd like to get rid of this implicit change on maxSize, and NOT allow `expireAfterWriteDuration` or `expireAfterAccessDuration ` to be set to `Duration.ZERO`.

But since this is out of scope for this PR feel free to ignore and I can do it in a separate PR :).